test/saithriftv2/Makefile

```diff
@@ -88,7 +92,7 @@ $(ODIR)/sai_rpc_server.o: ../../meta/sai_rpc_frontend.cpp
 	$(CXX) $(CPPFLAGS) -c $^ -o $@ $(CPPFLAGS) -I./gen-cpp -I../../meta -I../../inc -I../../experimental
 
 $(ODIR)/saiserver.o: src/saiserver.cpp $(CPP_SOURCES) directories
-	$(CXX) $(CPPFLAGS) -c src/saiserver.cpp  -o $@ $(CPPFLAGS) $(CDEFS) -I./gen-cpp -I../../inc
+	$(CXX) $(CPPFLAGS) -c src/saiserver.cpp  -o $@ $(CPPFLAGS) $(CDEFS) -I./gen-cpp -I../../inc -I../../experimental
```

Comment:
btw, this looks like an issue, as some part of the code is generated based on in tree headers and some are taken from /usr/include/sai. I think this needs to be fixed some time, not part of this change.

Reply:
If you think so, it might be good to file an issue. However, a later change could break automated workflows which use the current technique.